dsgrid/config/dimension_config.py

```diff
@@ -60,3 +60,35 @@ def get_unique_ids(self):
 
         """
         return {x.id for x in self.model.records}
+
+    def get_records(self):
```

Comment:
Why would you want this instead of the typed data models or a dataframe?

Reply:
You are right, the feature is already here: https://github.com/dsgrid/dsgrid/blob/1d4ecde7aadf73da1fae6e49b76c09e22c09e68c/dsgrid/dimension/dimension_records.py